app/models/species_list.rb

```diff
@@ -200,14 +218,18 @@ def name_included(name)
   end
 
   # After defining a location, update any lists using old "where" name.
+  # Original SQL:
+  # UPDATE species_lists
+  # SET `where` = #{new_name}, location_id = #{location.id}
+  # WHERE `where` = #{old_name}
   def self.define_a_location(location, old_name)
-    old_name = connection.quote(old_name)
-    new_name = connection.quote(location.name)
-    connection.update(%(
-      UPDATE species_lists
-      SET `where` = #{new_name}, location_id = #{location.id}
-      WHERE `where` = #{old_name}
-    ))
+    # Note: Need to use connection.quote_string here
+    old_name = SpeciesList.connection.quote_string(old_name)
+    new_name = SpeciesList.connection.quote_string(location.name)
+
+    SpeciesList.where(where: old_name).update_all(
+      where: new_name, location_id: location.id
+    )
```

Comment:
```suggestion
    SpeciesList.where(where: old_name).update_all(
      where: location.name, location_id: location.id
    )
```
I get easily confused by sql and Arel. But I think AR takes care of the quotes, and that `connection.quote_string` adds an extra, and therefore incorrect, level of quotes.
For example:
```sql
2.6.6 :001 > Observation.last.where
...
 => "Secret 'hidey hole', Oregon, USA" 
2.6.6 :002 > Observation.where(where: "Secret 'hidey hole', Oregon, USA")
  Observation Load (302.8ms)  SELECT  `observations`.* FROM `observations` WHERE `observations`.`where` = 'Secret \'hidey hole\', Oregon, USA' LIMIT 11
 => #<ActiveRecord::Relation [#<Observation id: 485741, created_at: "2022-05-09 23:51:49", updated_at: "2022-05-10 00:37:23", when: "2022-05-09", user_id: 4468, specimen: false, notes: {}, thumb_image_id: nil, name_id: 1, location_id: nil, is_collection_location: true, vote_cache: 0.0, num_views: 1, last_view: "2022-05-10 00:36:57", rss_log_id: 594018, lat: nil, long: nil, where: "Secret 'hidey hole', Oregon, USA", alt: nil, lifeform: " ", text_name: "Fungi", classification: "Domain: _Eukarya_", gps_hidden: false>]> 
2.6.6 :003 > Observation.where(where: Observation.connection.quote_string("Secret 'hidey hole', Oregon, USA"))
  Observation Load (288.7ms)  SELECT  `observations`.* FROM `observations` WHERE `observations`.`where` = 'Secret \\\'hidey hole\\\', Oregon, USA' LIMIT 11
 => #<ActiveRecord::Relation []> 
```
I will also shortly push a new test to cover this method. 
Please change/revert it if you don't like it.

Reply:
I like it.